### PR TITLE
initial kubeedge UI dashboard

### PR DESCRIPTION
**Pull request by SKYkubeedge**
Signed-off-by: SKYkubeedge <kubeedge@orientsky.com.cn>

### CODE REVIEW

**Comment by SKYkubeedge** [comment]
initial kubeedge UI dashboard

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+Install `node_modules`:
+
+```bash
+npm install
+```
+
+or
+
+```bash
+yarn
```
Comment (vincentgoat) [inline README.md:15]: Please correct the command execute order: first `yarn` and then `npm run start:dev`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+## Provided Scripts
+
+kubeedge provides some useful script to help you quick start and build with web project, code style check and test.
+
+Scripts provided in `package.json`. It's safe to modify or add additional script:
+
+### Start project
+
+正常启动请运行
+npm start:dev
```
Comment (vincentgoat) [inline README.md:27]: Should be `npm run start:dev`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+
+## Provided Scripts
+
+kubeedge provides some useful script to help you quick start and build with web project, code style check and test.
+
+Scripts provided in `package.json`. It's safe to modify or add additional script:
+
+### Start project
+
+正常启动请运行
```
Comment (vincentgoat) [inline README.md:26]: Please translate it into English.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+npm start:dev
+
+Mock测试模式请运行
```
Comment (vincentgoat) [inline README.md:29]: Please translate it into English.

**Pull request closed without merging**